microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -16,10 +17,12 @@
     TestSuiteMetadata,
     constants,
     simple_requirement,
+    CustomScript,
+    CustomScriptBuilder
 )
 from lisa.features import Gpu, GpuEnabled, SerialConsole
 from lisa.features.gpu import ComputeSDK
-from lisa.operating_system import AlmaLinux, Debian, Oracle, Suse
```

Comment:
We don't need to remove these. In you tests, you can simply add a check to run test if the images is Debian

Reply:
I think I mistakenly removed this line. Will be reverted.